src/slashcommands/Birthday.ts

```diff
@@ -151,14 +181,13 @@ export class Birthday implements SlashCommand {
 			//TODO: Permission Check
 
 			//ensure that the enmap has stored the user, and stores their birthday. If not create it and give it an empty string.
-			var userBirthday = birthdays.ensure(`${interaction.user.id}`, '');
+			var userBirthday = bdayDates.ensure(`${interaction.user.id}`, '');
```

Comment:
You aren't using userBirthday anywhere, there's no need to get the user's birthday before setting it. Especially if you're ensuring it to be an empty string. Also, stop interpolating strings like that. Use toString if need be.